src/pybind/mgr/requirements-mgr.in

```diff
@@ -0,0 +1,16 @@
+wheel~=0.31
+cherrypy~=18.8
+pyjwt~=2.4
+routes~=2.5
+werkzeug~=2.0
+saml~=0.7
+setuptools~=59.6
+setuptools_scm>=1.15
+importlib-metadata==4.8.0
+importlib-resources==5.4.0
+pkgconfig>=1.5
+zipp~=3.6
+typing-extensions~=4.1
+dataclasses~=0.6 # from werkzeug
+pecan~=1.4
+asyncssh~=2.9
```

Comment:
Suggestion: as we're now providing dependencies for multiple modules (dashboard, restful, etc.) and the list might keep growing, and since in ceph.spec it was really easy to relate deps to packages, what about splitting this into (only the min set required)?:
* `requirements-common.in`
* `requirements-mgr.in`
* `requirements-dashboard.in`
* `requirements-restful.in`

Alternatively, it could also work placing these deps under each subdir:
* `mgr/requirements.in`
* `mgr/dashboard/requirements.in`
* `mgr/restful/requirements.in`

The generated lockfile `requirements.txt` could be unique and lie under `mgr/`.